sql/2021/performance/web_vitals_by_rank.sql

```diff
@@ -0,0 +1,161 @@
+#standardSQL
+# Core WebVitals by rank
```

Comment:
Aside: These results are _really_ interesting. I'd love to see how the other CWV stats (per device/country/ECT) vary by rank as well!

Reply:
Device is covered in a very similar query in this PR! I can also look at the others but I might not report if it's not interesting.